internal/commands/config/list_keys_internal.go

```diff
@@ -0,0 +1,95 @@
+package config_internal
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingidentity/pingcli/internal/configuration"
+	"github.com/pingidentity/pingcli/internal/configuration/options"
+	"github.com/pingidentity/pingcli/internal/output"
+	"github.com/pingidentity/pingcli/internal/profiles"
+	"gopkg.in/yaml.v3"
+)
+
+func returnKeysYamlString() (string, error) {
+	var err error
+	validKeys := configuration.ViperKeys()
+
+	if len(validKeys) == 0 {
+		return "", fmt.Errorf("unable to retrieve valid keys")
+	}
+
+	validKeysJoined := strings.Join(validKeys, " ")
+	// Split the input string into individual keys
+	parts := strings.Split(validKeysJoined, " ")
+	result := make(map[string]interface{})
+
+	// Iterate over each part
+	for _, part := range parts {
+		// Skip the "activeProfile" key
+		if part == "activeProfile" {
+			continue
+		}
+
+		// Create a nested map for each part
+		currentMap := result
+		keys := strings.Split(part, ".")
+		for i, key := range keys {
```

Comment:
Just a nit, and can be ignored, but I think more clarity on variable names for `keys`, `key`, `parts` and `part` help make this yaml creation algorithm more readable